### DISCUSSION TITLE
Unable to delete devices to due 'django.db.utils.IntegrityError'

### DISCUSSION THREAD

**Discussion by jcralbino** [netbox-community/netbox#6279] [Q&A] [answered]
Hello 

I have installed previously the ntc-netbox-plugin-onboarding to test, and after realizing that this was not going to help to onboard HPE devices due to broken napalm libraries i removed this plugin.

Now and after a few upgrades, i am getting this error:
```
<class 'django.db.utils.IntegrityError'>

update or delete on table "dcim_device" violates foreign key constraint "netbox_onboarding_on_device_id_cee3ef7b_fk_dcim_devi" on table "netbox_onboarding_onboardingdevice"
DETAIL:  Key (id)=(261) is still referenced from table "netbox_onboarding_onboardingdevice".


Python version: 3.6.8
NetBox version: 2.11.0
```

I have several objects that i cannot delete now.
How can i fix this?

**Comment by jeremystretch** [comment, marked as answer]
Hey @jcralbino, you should be able to just drop the `netbox_onboarding_onboardingdevice` PostgreSQL table since you're not using the plugin anymore.

**Reply by jcralbino** [reply]
Will test that then after taking a vm snapshot just in case

**Reply by jcralbino** [reply]
Dropped the table and it is ok now

Thanks
